cde-core/src/pt/webdetails/cdf/dd/model/inst/writer/cdfrunjs/dashboard/amd/PentahoCdfRunJsDashboardWriteContext.java

```diff
@@ -33,14 +36,16 @@ public PentahoCdfRunJsDashboardWriteContext( CdfRunJsDashboardWriteContext facto
 
   @Override
   public String replaceTokens( String content ) {
+    final long timestamp = this._writeDate.getTime();
     final String path = this._dash.getSourcePath().replaceAll( "(.+/).*", "$1" );
+    final String root = getRoot();
 
-    return content
+    String result = content
```

Comment:
I think this extra String pointer is not necessary, can you just `return content.replaceAll(...)`?